securedrop/tests/test_source.py

```diff
@@ -21,7 +22,7 @@
 from .utils.db_helper import new_codename
 from .utils.instrument import InstrumentedApp
 
-overly_long_codename = 'a' * (Source.MAX_CODENAME_LEN + 1)
+overly_long_codename = 'a' * (LoginForm._MAX_CODENAME_LEN + 1)
```

Comment:
Just have to point out again that I don't think `LoginForm` should be the source of truth for maximum codename length. I can be so tiresome. I apologize. :slightly_frowning_face: